src/chart/treemap/Breadcrumb.ts

```diff
@@ -185,7 +190,11 @@ class Breadcrumb {
                 onclick: curry(onSelect, itemNode)
             });
             (el as ECElement).disableLabelAnimation = true;
-
+            el.getTextContent().ensureState('emphasis').style = createTextStyle(emphasisTextStyleModel, { text });
+            el.ensureState('emphasis').style = emphasisItemStyle;
```

Comment:
Now multiple elements are sharing the same `emphasisItemStyle`, will it bring any side effects?

Reply:
All `breadcrumb` item share same `emphasis` style, there is no way to set each item their own `emphasis` style. So I think It should be ok to use same style instance